test/PDPVerifier.t.sol

```diff
@@ -1542,63 +1513,68 @@ contract PDPListenerIntegrationTest is Test {
     }
 
     function testListenerPropagatesErrors() public {
-
-        // Can't create a proof set with a bad listener
+        // CREATE operation
```

Comment:
The new comments are not adding enough clarity to be worth keeping.  `OperationType.CREATE` makes it clear which operation we're dealing with from reading the code.  But removing the old comment is fine -- it also wasn't adding enough clarity to be worth it.